addon/src/modifiers/css-transition.js

```diff
@@ -1,309 +1,631 @@
 import Modifier from 'ember-modifier';
+import { registerDestructor } from '@ember/destroyable';
+import { macroCondition, dependencySatisfies } from '@embroider/macros';
```

Comment:
I think `@embroider/macros` still needs to be added as a dependency in `package.json`?

Reply:
for some reason, I was under impression there is no need to explicitly list `@embroider/macros` in `dependencies`.
Will check confirm if it's the case.

Anyway, this is not necessary from the perspective that `@embroider/macros` always available for the v2 addons because v2 addons get consumed via `ember-auto-import` which does provide those deps.

Reply:
I would think that if you directly use a package, it's best to define it as a dependency regardless of the fact if it's also being provided by some other package. How else can you be sure that you are using the right version?

Reply:
@bertdeblock totally valid point! Added `@embroider/macros` to `dependencies`

Reply:
@bertdeblock just FYI I found where I seen this - https://github.com/embroider-build/embroider/blob/main/PORTING-ADDONS-TO-V2.md?plain=1#L207-L214

Reply:
Good to know, didn't know that, thanks!